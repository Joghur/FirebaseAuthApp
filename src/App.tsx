/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * Generated with the TypeScript template
 * https://github.com/react-native-community/react-native-template-typescript
 *
 * @format
 */

import React from 'react';
import {SafeAreaView, ScrollView, StyleSheet, Text, View} from 'react-native';

const App = () => {
  return (
    <SafeAreaView>
      <ScrollView contentInsetAdjustmentBehavior="automatic">
        <View>
          <Text style={styles.highlight}>App.tsx</Text>
        </View>
      </ScrollView>
    </SafeAreaView>
  );
};

const styles = StyleSheet.create({
  highlight: {
    fontWeight: '700',
  },
});

export default App;
