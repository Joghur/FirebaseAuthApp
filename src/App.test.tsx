import {render} from '@testing-library/react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

it('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Text element', () => {
  const {getByText} = render(<App />);
  expect(getByText('App.tsx').props.children).toBe('App.tsx');
});
