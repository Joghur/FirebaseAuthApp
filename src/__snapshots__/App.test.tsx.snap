// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <RCTScrollView
    contentInsetAdjustmentBehavior="automatic"
  >
    <View>
      <View>
        <Text
          style={
            Object {
              "fontWeight": "700",
            }
          }
        >
          App.tsx
        </Text>
      </View>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
